.travis.yml

```diff
@@ -27,7 +27,7 @@ before_install:
 # command to install dependencies
 install:
     - pip install -r requirements.txt
-    - pip install .[extreme_scale]
+    - pip install .[extreme_scale] .[monitoring]
```

Comment:
This should maybe be phrased as `.[extreme_scale,monitoring]`

Reply:
fixed